### ISSUE TITLE
Items not visible in JEI or creative tab 1.12.2

### ISSUE THREAD

**Issue by Liiw** [cam72cam/ImmersiveRailroading#224]
Hello!

**Versions of stuff installed:**

MC: 1.12.2
Forge: 14.23.2.2618
ImmEng: 0.12-76
ImmRail: 0.5.0_1.12
JEI: 1.12.2-4.8.5.151
JEI integration: 1.12.2-1.5.1.36

also on the server there is SpongeForge 1.12.2-2611-7.1.0-BETA-2925

**What the issue is**

When playing on a SMP server searching for items from immersive railroading, there is nothing to be found in JEI. I also changed my gamemode to creative to take a look at the IR tabs, but they are also empty:

![image](https://user-images.githubusercontent.com/14122419/36504394-0ccfebba-1759-11e8-8fae-df92340f4167.png)

This is also the case with other people on the server who do not have the texture pack, I have, installed

**Comment by cam72cam** [comment]
Can you try a different JEI version?  I am not seeing this problem anywhere else?  You are running a beta release of it.

**Comment by cam72cam** [comment]
@Liiw if you upgrade or downgrade JEI and are still experiencing this issue, let me know and I will re-open this issue

**Comment by Liiw** [comment]
@cam72cam I performed a pack update. Also updated JEI to the latest version. Tested it, it did not work. Then I downgraded JEI to the last Release version, No luck. Then I even tried the server without Sponge, that was also not the case. 

While lowering the JEI version to last years Release version, I noticed all is not proper  IR versions on the client and server. I messed up the version numbers on server and client. On the client I had "0.5.0_1.12" while on the server there was "0.4.5a_1.12". This was the problem because after updating both versions to the latest and also reverting back to the latest JEI version, I can properly see all the items in JEI. 

Going more deep into this issue as to why I would make such a simple mistake. I am using the Twitch program to manage two different profiles- one for the server and other for the client. While the client version updated all the mods properly, IR's "0.4.5a" version did not notify me of an update possibility on the server profile. After I found the mistake, I even tried to manually click on it and update it from there, but it would no open. After updating the mod by copying it to the profile folder, it started working again. Might be the program, might be my mistake, I'm not sure anymore.

Anyway, long story short. Thank you for your time! All is working great again!

**Comment by cam72cam** [comment]
Phew, thanks for the update!
